nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -60,6 +60,7 @@ let
       inherit (efi) canTouchEfiVariables;
       inherit (cfg)
         version extraConfig extraPerEntryConfig extraEntries forceInstall useOSProber
+        extraGrubInstallArgs
```

Comment:
This style is nice for keeping the diff clean.  But I would prefer that the lines were similar lengths, or if similar things were grouped together.

Reply:
I need to optimise for a clean, minimal diff here, as I need to cherry-pick this into various projects / backports and need to avoid merge conflicts.

If we want to do extra grouping, I think it should be done in a separate PR.